### ISSUE TITLE
Reenable support for MOS jobs

### ISSUE THREAD

**Issue by DavidHuber-NOAA** [NOAA-EMC/global-workflow#3196]
### What new functionality do you need?

The MOS jobs should be able to run on WCOSS2.  The current version of MOS (v5.4.3) uses the old `COM` structure and needs to be updated as part of this exercise.

### What are the requirements for the new functionality?

All `MOS` jobs are able to produce expected products.

### Acceptance Criteria

Enabling `DO_MOS=YES` generates, archives, and produces HPSS tarballs of MOS data products when run on WCOSS2.

**Comment by DavidHuber-NOAA** [comment]
@aerorahul can we close this issue?

**Comment by aerorahul** [comment]
yeah, we should.  I doubt we will be able to support MOS in the global-workflow.
